libexec/phpenv-configure

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# Summary: Edit the current PHP's php.ini configuration file at location
+
+set -e
+[ -n "$PHPENV_DEBUG" ] && set -x
+
+vi "$(php --ini|grep Loaded|awk '{print $NF}')"
```

Comment:
rather `${EDITOR:-vi}`

Reply:
I'll look into this. Not sure which is more "standard" between `editor` (executable) and `$EDITOR` (variable) as I'm not sure that `$EDITOR` is always defined.

Reply:
Suggestion integrated into #87